tests/common.py

```diff
@@ -1953,3 +1954,26 @@ def get_schema_suggested_value(schema: vol.Schema, key: str) -> Any | None:
                 return None
             return schema_key.description["suggested_value"]
     return None
+
+
+class MockDhcpServiceInfo(DhcpServiceInfo):
```

Comment:
Should we put it in `tests.service_info` (or something else) if more are planned?

Reply:
> Should we put it in `tests.service_info` (or something else) if more are planned?

Makes sense...